pallet-logion-loc/src/tests.rs

```diff
@@ -185,32 +185,6 @@ fn it_fails_adding_metadata_when_closed() {
 	});
 }
 
-#[test]
-fn it_fails_adding_metadata_on_polkadot_transaction_loc_when_submitter_is_invalid() {
```

Comment:
What about keeping the tests but just expecting they succeed?

Reply:
Agreed ! We never have too much tests.